src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -990,9 +990,11 @@ public void SetCookies(Uri uri, string cookieHeader)
         }
     }
 
+    // PathList needs to be public in order to maintain binary serialization compatibility as the System shim
+    // needs to have access to type-forward it.
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    internal struct PathList
+    public struct PathList
```

Comment:
While it is true it isn't in the ref and we don't expect folks to directly access it we may still have limitations on what we can change in the future, primarily because of the serialization compat. If we feel we need to change it to a class I would prefer we do it as part of making it public to avoid potential compat issue.